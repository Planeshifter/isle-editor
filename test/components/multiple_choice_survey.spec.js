// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import MultipleChoiceSurvey from '@isle-project/components/multiple-choice-survey/main.js';


// TESTS //

describe( '<MultipleChoiceSurvey />', function test() {
	it( 'renders a survey', () => {
		const survey = renderer.create(
			<MultipleChoiceSurvey />
		).toJSON();
		expect( survey ).toMatchSnapshot();
	});
});
