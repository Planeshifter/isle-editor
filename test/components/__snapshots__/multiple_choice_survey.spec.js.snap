// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultipleChoiceSurvey /> renders a survey 1`] = `null`;
